docs/docs/integrations/shopify.mdx

```diff
@@ -0,0 +1,62 @@
+# Elevate Your Shopify Store with GrowthBook Experiments in Just 3 Simple Steps!
+
+Unleash the power of experimentation with GrowthBook to supercharge your Shopify store—no coding skills required!
+
+:::info Note
+
+This guide walks you through creating experiments using the GrowthBook Visual Editor, which requires a `Pro` subscription. [Learn More](https://www.growthbook.io/pricing).
+
+:::
+
+## Let's Get Started
+
+### Step 1: Create a GrowthBook SDK Connection
+
+To connect your GrowthBook account to Shopify, you'll need to create a new SDK Connection and select `Javascript` as the language. Before saving, confirm that you've enabled the toggles for `Include visual experiments in endpoint's response?` and `Include draft experiments`.
+
+![](../../static/images/integrations/shopify/growthbook-shopify-sdk-connection.png)
+
+### Step 2: Add GrowthBook to Your Shopify Theme
+
+Once the SDK Connection is created, you'll want to locate the `Client Key` from the <strong>Implementation Instructions</strong> section.
+
+Then, log into your Shopify account, navigate to your online store, and edit your store's code.
+
+![](../../static/images/integrations/shopify/growthbook-shopify-edit-code.png)
+
+Once you're in the code editor, locate the `theme.liquid` file and add the following snippet to the file just before the closing `</head>` tag. Be sure to replace the `data-client-key` with the `Client Key` you located above.
+
+```js
+    <script id="growthbook-sdk" src="https://cdn.jsdelivr.net/npm/@growthbook/growthbook/dist/bundles/index.min.js" defer></script>
+
+    <script async
+      data-client-key="ADD_YOUR_CLIENT_KEY_HERE"
+      src="https://cdn.jsdelivr.net/npm/@growthbook/growthbook/dist/bundles/auto.min.js"
+    ></script>
+```
+
+:::info Note
+
+Using the code snippet above automatically tracks experiment exposures with both GA4 (using dataLayer) and Segment (using analytics.track) when available. [Follow our guide here](https://docs.growthbook.io/app/visual#option-2-pre-built-script-tag-best-for-shopify-webflow) if you want to use a different analytics provider.
+
+:::
+
+Once added, click "Save" and navigate back to GrowthBook.
+
+### Step 3: Create a GrowthBook Visual Editor Experiment
+
+Now that GrowthBook is installed on your Shopify store, you can create a new experiment using the Visual Editor.
+
+First, navigate to "Experiments" on the left-hand navigation menu and then click "Create Experiment", before selecting "Design a New Experiment" and following the on-screen prompts.
```

Comment:
Is this assuming that they have already installed the Chrome Extension? We should link to those instructions just in case.

https://docs.growthbook.io/app/visual#requirements

Reply:
Good call. I'll add the same to the Webflow section.